bait-files/src/main/java/com/google/sps/servlets/UserHistoryServlet.java

```diff
@@ -64,8 +51,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     List<Review> reviews = allRequests.stream()
                                .filter(rev -> rev.userId.equals(userId))
                                .filter(rev -> rev.status == statusType)
```

Comment:
Have you pulled from the master branch? There were a lot of changes to this servlet that were merged. This seems to be from before this was refactored.

Reply:
Just pulled